src/Sulu/Bundle/AdminBundle/Resources/js/containers/Link/tests/overlays/ExternalLinkTypeOverlay.test.js

```diff
@@ -84,6 +84,27 @@ test('Pass correct props to Dialog', () => {
     expect(externalLinkOverlay.find('Dialog').prop('open')).toEqual(false);
 });
 
+test('Display given URL with query parameters in href input', () => {
+    const targetChangeSpy = jest.fn();
+    const urlChangeSpy = jest.fn();
+
+    const externalLinkOverlay = shallow(
+        <ExternalLinkTypeOverlay
+            href="http://www.sulu.io?param=value-1"
+            onCancel={jest.fn()}
+            onConfirm={jest.fn()}
+            onHrefChange={urlChangeSpy}
+            onTargetChange={targetChangeSpy}
+            onTitleChange={jest.fn()}
+            open={true}
+            target="_blank"
+            title={undefined}
+        />
+    );
+
+    expect(externalLinkOverlay.find('Url').prop('value')).toEqual('http://www.sulu.io?param=value-1');
```

Comment:
```suggestion
    expect(externalLinkOverlay.find('Url').prop('value')).toEqual('http://www.sulu.io/contact-us?param=value-1');
```

Reply:
pushed the change together with linting fixes 🙂